device/mellanox/x86_64-mlnx_msn3800-r0/sensors.conf

```diff
@@ -8,6 +8,38 @@
 bus "i2c-2" "i2c-1-mux (chan_id 1)"
     chip "mlxsw-i2c-*-48"
         label temp1 "Ambient ASIC Temp"
+        ignore temp2
```

Comment:
There are 64 ports in 2740, but there are 32 ignore here

Reply:
fixed